File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        count = [0]*26
        for i in range(len(s)):
            count[ord(s[i]) - ord("a")] += 1
        for j in range(len(t)):
            count[ord(t[j]) - ord("a")] -= 1
        for i in range(len(count)):
            if count[i] != 0:
                return False
        return True

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int count[] = new int[26];
        for(int i=0; i<s.length(); i++){
            count[s.charAt(i)-'a'] += 1;
        }
        for(int j=0; j<t.length(); j++){
            count[t.charAt(j) - 'a'] -= 1;
        }
        for(int k=0; k<count.length; k++){
            if (count[k] != 0){
                return false;
            }
        }
        return true;
    }
}